apps/ergw_core/src/ergw_core_config.erl

```diff
@@ -36,7 +36,21 @@ to_map(L) when is_list(L) ->
 	      erlang:error(badarg, [Opt])
       end, #{}, normalize_proplists(L)).
 
-get([Key|Next], Default) ->
+get([Key|_] = Query, Default) ->
+    case is_global_key(Key) of
+	true  -> get_global_key(Query, Default);
+	false -> get_local_key(Query, Default)
+    end.
+
+get_global_key(Query, Default) ->
+    case ergw_global:find(Query) of
+	{ok, Value} ->
+	    {ok, Value};
+	_ ->
+	    {ok, Default}
```

Comment:
This function will return value in any case, maybe make sense return only value instead of tuple, eg like in [get_env/3](https://erlang.org/doc/apps/kernel/application.html#get_env-3)?

Reply:
the {ok, _} return is preparation of also returning error later, e.g. it might be better to return an error when the raft handler fails.